<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>entryView</class>
 <widget class="QDialog" name="entryView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>250</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="title">
      <string>E: {}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="2">
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <widget class="QPushButton" name="load">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Load</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="remove">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>remove</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QTreeWidget" name="treeWidget">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>150</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>150</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::Box</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustToContents</enum>
               </property>
               <property name="dragEnabled">
                <bool>true</bool>
               </property>
               <property name="animated">
                <bool>true</bool>
               </property>
               <column>
                <property name="text">
                 <string>Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Config</string>
                </property>
               </column>
               <item>
                <property name="text">
                 <string>get</string>
                </property>
                <item>
                 <property name="text">
                  <string>host</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>port</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>user</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>hostKey</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>useHostKey</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>sources</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>destination</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>remote_home</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>keyEncrypted</string>
                 </property>
                </item>
               </item>
               <item>
                <property name="text">
                 <string>send</string>
                </property>
                <item>
                 <property name="text">
                  <string>host</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>port</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>user</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>hostKey</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>useHostKey</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>sources</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>destination</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>remote_home</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>keyEncrypted</string>
                 </property>
                </item>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
